get_openai_responses.py

```diff
@@ -0,0 +1,50 @@
+import json
+from PIL import Image
+from pathlib import Path
+from openai_api import OpenAI_API
+from ocr import OCRProcessor
+from ocr_enum import OCREngine
+from image_diff import image_crop_dialogue_box
+
+ocr = OCRProcessor(OCREngine.OPENAI)
+openai = OpenAI_API(vision_model="gpt-4o")
+
+# Specify the directories for input images and outputs
+images_folder = Path('../data/data_en/')
+output_file = Path('../data/ocr_responses_en_gpt4o.json')
+dialogue_box_folder = Path('../data/dialogue_boxes_en_gpt4o/')
+
+# Create the dialogue box folder if it doesn't exist
+dialogue_box_folder.mkdir(parents=True, exist_ok=True)
+
+ocr_responses = {}
+
+# Iterate over all image files in the specified folder
+for image_path in images_folder.glob('*.jpg'):
+    try:
```

Comment:
can we make like an openai package or something so we don't have new files in root

Reply:
These scripts are for generating some test samples. They are not really needed anymore. I'll keep them in a scripts folder.